readme.md

```diff
@@ -1,7 +1,7 @@
 # Product cards
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://MelnychukVol.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://MelnychukVol.github.io/layout_product-cards/report/html_report/)
```

Comment:
please add these links to Pull Request

Reply:
isn't fixed

Reply:
https://melnychukvol.github.io/layout_product-cards/report/html_report/
https://melnychukvol.github.io/layout_product-cards/

Reply:
You should add these links here 
![image](https://github.com/mate-academy/layout_product-cards/assets/64750958/20c1ea51-8538-4e5a-863e-4aff8b2fd35b)